doc/nrf/includes/vsc_build_and_run_nrf53.txt

```diff
@@ -0,0 +1,14 @@
+1. Open Visual Studio code.
+#. Click on the :guilabel:`nRF Connect for VS Code` icon in the left navigation bar and then click :guilabel:`Open`.
+#. Click on :guilabel:`Add an existing application`.
```

Comment:
Hm, I think this step is incorrect? I tried to follow this step myself and it did not work.
Instead, I think it should be this instead, with some additional steps:

```suggestion
#. Click on :guilabel:`Create a new application from sample`.
#. For `Application type`, click on "Freestanding"
#. In the prompt, navigate to the folder...
#. Click on `Create Application`
    You should now see the selected application...
```

![image](https://user-images.githubusercontent.com/51916561/168276127-dfdb8d43-56e1-456f-9a19-93e1e184a68d.png)

Reply:
My understanding is that **Add an existing application** is for when you want to continue after creating an app (clicked **Create Application**) but didn't save your work, so it's not in the list of applications but does exist in the folder you chose for **Application location**.

Reply:
Yes, that is right. 
So, if this text is used in a context where the user has already built this sample previously, then it is fine as it (I believe).

Reply:
Yes, The understanding is that we are selecting any existing applications/samples (highlighted in yellow). 
![image](https://user-images.githubusercontent.com/91891495/168580435-932b7eca-af3a-42a9-92e4-34b570772b2d.png)

Reply:
I was looking at the VSC documentation, and they use **Create a new application from sample** for the basic instructions.

Reply:
Both ` Add an existing application` and `Create a new application from sample` can be used to initialize application.  The difference is that` Create a new application from sample` copies sample to the new folder in the root of workspace and  ` Add an existing application` uses original version of ample.

Reply:
Okay, in that case it is fine as is. Thanks for the explanation.
I will mark this as resolved now.